src/styles/BookList.css

```diff
@@ -25,12 +27,13 @@
 }
 
 .list-item {
+  align-items: center;
+  background-color: #fff;
+  box-shadow: 0 0 15px 10px rgba(238, 239, 243, 0.8);
   display: flex;
-  justify-content: space-between;
-  padding: 40px;
-  background-color: white;
-  border: 0.5px solid rgb(210, 181, 181);
-  margin: 11px 0;
+  gap: 10%;
+  margin-bottom: 20px;
+  padding: 15px 20% 15px 45px;
 }
 
```

Comment:
- [ ] Kindly align the text `ADD NEW BOOK` to the left as per the design. Please also make sure you are using the correct color for the text. 🥸️

**Your design**
![Screenshot from 2023-03-11 11-41-44](https://user-images.githubusercontent.com/17243325/224476152-feb4d47c-935f-46bc-96ff-82b47aef85a0.png)

**Zeplin design**
![Screenshot from 2023-03-11 11-41-34](https://user-images.githubusercontent.com/17243325/224476177-d7e53c82-1320-4415-a198-f9449b49c3b4.png)